src/index.html

```diff
@@ -1,12 +1,352 @@
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__content">
+    <header class="header">
+      <div class="logo">
+        <a href="#Home">
+          <img
+            src="./images/logo.png"
+            alt="logoMoyo"
+          >
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#Apple"class="nav__link--is-active nav__link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Samsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Smartphones" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#Laptops&#38;Computers"
+              data-qa="nav-hover"
+              class="nav__link"
+            >
+            Laptops &#38; Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Gadgets" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Tablets" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Photo" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Video" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="page__main">
+      <section data-qa="card" class="card page__card" >
+        <img
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+          class="card__logo"
+        >
+
+        <h2 class="card__description">
+           APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__stars stars">
+          <div class="stars__container stars__container--4">
+            <div class="stars__container-star"></div>
+            <div class="stars__container-star"></div>
+            <div class="stars__container-star"></div>
+            <div class="stars__container-star"></div>
+            <div class="stars__container-star"></div>
+          </div>
+
+          <p class="stars__reviews-text"> Reviews: 5</p>
+        </div>
+
+        <div class="card__prices">
+          <p class="card__text">Price:</p>
+          <p class="card__amount">$2,199</p>
+        </div>
+
+        <a
+        data-qa="card-hover"
+        class="card__link-buy"
+        href="/"
+        >
```

Comment:
please pay attention to distances
```suggestion
        <a
          data-qa="card-hover"
          class="card__link-buy"
          href="/"
        >
```